lager/deps.hpp

```diff
@@ -334,12 +334,17 @@ class deps
     /*!
      * Creates a `deps` object picking the dependencies from `other`.
      */
+#ifdef LAGER_DISABLE_STORE_DEPENDENCY_CHECKS
+    template <
+        typename... Ds>
+#else
     template <
         typename... Ds,
         std::enable_if_t<required_key_set == boost::hana::intersection(
-                                                 required_key_set,
-                                                 deps<Ds...>::required_key_set),
-                         bool> = true>
+            required_key_set,
+            deps<Ds...>::required_key_set),
+        bool> = true>
+#endif
```

Comment:
Nitpick: I may slightly prefer this form:
```c++
    template <
        typename... Ds
#ifndef LAGER_DISABLE_STORE_DEPENDENCY_CHECKS
       , std::enable_if_t<required_key_set == boost::hana::intersection(
                                                 required_key_set,
                                                 deps<Ds...>::required_key_set),
                         bool> = true>
            required_key_set,
            deps<Ds...>::required_key_set),
        bool> = true
#endif
    >
```
Same below.

Reply:
This is fixed.